components/firecrawl/firecrawl.app.mjs

```diff
@@ -1,11 +1,190 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "firecrawl",
-  propDefinitions: {},
+  propDefinitions: {
+    url: {
+      type: "string",
+      label: "URL",
+      description: "The URL to start crawling from.",
+    },
+    crawlId: {
+      type: "string",
+      label: "Crawl ID",
+      description: "Identifier of a certain crawl operation.",
+    },
+    includes: {
+      type: "string[]",
+      label: "Includes",
+      description: "URL patterns to include.",
+    },
+    excludes: {
+      type: "string[]",
+      label: "Excludes",
+      description: "URL patterns to exclude.",
+    },
+    generateImgAltText: {
+      type: "boolean",
+      label: "Generate Image Alt Text",
+      description: "Generate alt text for images using LLMs (must have a paid plan).",
+    },
+    returnOnlyUrls: {
+      type: "boolean",
+      label: "Return Only URLs",
+      description: "If true, returns only the URLs as a list on the crawl status. Attention: the return response will be a list of URLs inside the data, not a list of documents.",
+    },
+    maxDepth: {
+      type: "string",
```

Comment:
Wouldn't it be better if this was an `integer` instead? From the description, it doesn't seem fractional values are allowed

Reply:
yes but it accepts zero as a value and integer props do not